db/schema.rb

```diff
@@ -20,7 +20,7 @@
     t.uuid "record_id", null: false
```

Comment:
I don't think there are any schema changes so do we need to commit this file as part of this pr? (we seem to keep cycling between adding and removing the precision: nils)